spec/lib/evss/pciu/request_body_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe EVSS::PCIU::RequestBody do
+  let(:phone) { build(:phone_number, :nil_effective_date) }
+  let(:now) { '2018-04-02T14:02:59+00:00' }
+
+  describe '#set' do
+    before { Timecop.freeze now }
+    after { Timecop.return }
```

Comment:
I believe this Timecop manipulation could be done just using a `run_at:` call, similar to here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/middleware/rack_attack_spec.rb#L39

Reply:
@rlbaker - I just tried it, and that works.  The issue I am running into is I use that `now` value a couple times in the spec.

Based on the compiling, I can't use `run_at:` in conjunction with anything that is not a hardcoded string.  So I would have to replace where `now` is being used with `'2018-04-02T14:02:59+00:00'` each time.  So there would be a little bit of a "magic number" situation.

I tried this with setting `now` in a `let`, an instance variable (with is no bueno but wanted to see if it worked :) ),  as well as a dedicated support method.  It comes up as undefined in these cases.

What are your thoughts?  Any other ideas?

Reply:
@hpjaj I think in this case then it's fine to do what you're doing. Just wanted to present an alternative that might work :) Sorry it didn't!

Reply:
Me, too.  That is a good idea...gonna remember that one!